deeppavlov/models/doc_retrieval/logit_ranker.py

```diff
@@ -70,6 +72,6 @@ def __call__(self, contexts_batch: List[List[str]], questions_batch: List[List[s
                 results = sorted(results, key=lambda x: (x[0] != '', x[2]), reverse=True)
             else:
                 results = sorted(results, key=itemgetter(2), reverse=True)
-            best_answer = results[0][0]
-            batch_best_answers.append(best_answer)
-        return batch_best_answers
+            batch_best_answers.append(results[0][0])
+            batch_best_answers_scores.append(results[0][2])
```

Comment:
It is useful, as in it makes it a skill. If confidence is between 0 and 1.

Reply:
Yes, confidence is in [0, 1].

I don't think that returning only top scored answer is not useful to adjust threshold. How other answers can help with it? (their scores will be always lower than top 1 one)

Reply:
Returning only 1 score is better than nothing, but returning `top_n` scores is more coherent with current rankers architecture and more useful when testing the ODQA quality (in my experimental branches this class returns `top_n`) and is not hard to implement.
It also may be useful in adjusting the threshold, depends on the adjusting algorithm. For example, if you need to take first `top_n` cumulative score for adjusting. Usually, when the reader returns a correct true answer, top 2nd and top 3d answers are similar to 1d.

Reply:
It can be done as new feature in following PRs.

Reply:
OK, I'm making a PR with this new feature.

Reply:
And warning about the new feature in this PR after tests pass.

Reply:
Actually, no need in warning for `top_n` in `logit_ranker`, since the API doesn't change. In both cases, it returns 2 lists, with answers and scores. But without `top_n` both lists just have len=1.